app/views/alchemy/essences/_essence_boolean_editor.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="content_editor essence_boolean" id="<%= essence_boolean_editor.dom_id %>" data-content-id="<%= essence_boolean_editor.id %>">
+<div class="content_editor essence_boolean <%= essence_boolean_editor.css_classes %>" id="<%= essence_boolean_editor.dom_id %>" data-content-id="<%= essence_boolean_editor.id %>">
```

Comment:
With the new `css_classes` helper we can remove the other classes from the template and change this to

```suggestion
<%= content_tag :div, class: essence_boolean_editor.css_classes, id: essence_boolean_editor.dom_id, data: {content: {id: essence_boolean_editor.id, name: essence_boolean_editor.name}} %>
```